### DISCUSSION TITLE
default template TLS_server is missing from 2.8.0

### DISCUSSION THREAD

**Discussion by wenzhuoz** [chris2511/xca#597] [General]
I just updated xca to 2.8.0 using flatpak. It looks that the useful default template TLS_server is now gone.

**Comment by t2mercer** [comment]
The .xca files are located here: https://github.com/chris2511/xca/tree/main/misc

[CA.xca](https://github.com/chris2511/xca/blob/main/misc/CA.xca)
[TLS_client.xca](https://github.com/chris2511/xca/blob/main/misc/TLS_client.xca)
[TLS_server.xca](https://github.com/chris2511/xca/blob/main/misc/TLS_server.xca)

Click on Template tab at top and then import the three templates
